README.md

```diff
@@ -1,4 +1,6 @@
-# companion-module-spx-gc
+# companion-module-spx
 See [HELP.md](./HELP.md) and [LICENSE](./LICENSE)
 
-API found here: [http://127.0.0.1:5000/api/v1]
\ No newline at end of file
+This is a fork from [companion-module-spx-gc](https://github.com/bitfocus/companion-module-spx-gc) to support Companion 3.x
```

Comment:
This line doesnt look appropriate once merged

Reply:
see latest commit